downstream/modules/platform/con-gs-final-set-up.adoc

```diff
@@ -1,10 +1,10 @@
 [id="con-gs-final-set-up"]
 
-= Final setup and usage 
+= Using this guide
 
 After you have installed {PlatformNameShort} 2.5 and have become familiar with the dashboard, use this document to explore further options for setup and daily use. 
```

Comment:
suggest using the attribute {PlatformVers} instead of 2.5.

Reply:
updated, ty